apps/tailwind-components/components/form/Legend.vue

```diff
@@ -19,9 +19,11 @@
           @click.prevent="emit('goToSection', section.id)"
         >
           <span
+            v-if="section.id != '_mg_top_of_form'"
             class="text-title-contrast capitalize"
             :class="{
               'font-bold': section.isActive,
+              'ml-2 italic': hasSections && section.type === 'HEADING',
             }"
           >
```

Comment:
In terms of the html markup, sections should be structured as a nested list.
 
```html
<ul>
    <li>Heading</li>
    <li>Section</li>
    <ul>
        <li>Section Heading</li>
    </ul>
</ul>
```

I would probably recommend switching to an ordered list as there's a specific order to the headings.

Reply:
I find it a bit distracting to italicize the section label. It would be good to make the style consistent with the current headings. I think increasing the margin a bit more would help differentiate between headings and sections (e.g., `ml-4`).